.github/workflows/codecov.yaml

```diff
@@ -0,0 +1,35 @@
+name: Codecov workflow
+on: [push]
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
This makes the workflow to run for each python version. Wondering if this is needed at this point 🤔

Reply:
It's good to know, if something specific like bug in code will be connected with particular Python version and OS. This CI gives us red flag, so I think we no need additional multiplatform verifications for unit tests.